lib/options.json

```diff
@@ -134,10 +134,14 @@
       ]
     },
     "hot": {
-      "type": "boolean"
-    },
-    "hotOnly": {
-      "type": "boolean"
+      "anyOf": [
+        {
+          "type": "boolean"
+        },
+        {
+          "enum": ["only"]
```

Comment:
I'm not sure I like this change to the hot option because webpack core CLI also has `--hot` boolean flag (https://webpack.js.org/api/cli/) and I don't think it's great for usage of a flag of the same name to diverge.

Reply:
@Loonride we should implement `hot: only` on webpack-cli side too

Reply:
The main problem is that we already have a lot of options and setting up everything is terribly complicated and redundant, we must simplify everything as much as possible

Reply:
And in some edge cases it is normal what package have same arguments but with many values